src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -0,0 +1,828 @@
+                    indexOffset: 0,
+                    isDisabled: canModifyParticipantsValue,
+                },
+                {
+                    title: translate('moneyRequestConfirmationList.splitWith'),
+                    data: formattedParticipantsList,
+                    shouldShow: true,
+                    indexOffset: 1,
+                },
+            );
+        } else {
+            const formattedSelectedParticipants = selectedParticipants.map((participant) => ({
+                ...participant,
+                isDisabled: ReportUtils.isOptimisticPersonalDetail(participant.accountID),
+            }));
+            sections.push({
+                title: translate('common.to'),
+                data: formattedSelectedParticipants,
+                shouldShow: true,
+                indexOffset: 0,
+            });
+        }
+        return sections;
+    }, [
+        selectedParticipants,
+        hasMultipleParticipants,
+        iouAmount,
+        iouCurrencyCode,
+        getParticipantsWithAmount,
+        payeePersonalDetailsMemo,
+        translate,
+        canModifyParticipantsValue,
+        selectedParticipantsMemo,
+    ]);
+
+    const selectedOptions = useMemo(() => {
+        if (!hasMultipleParticipants) {
+            return [];
+        }
+        const myIOUAmount = IOUUtils.calculateAmount(selectedParticipantsMemo.length, iouAmount, iouCurrencyCode ?? '', true);
+        return [...selectedParticipantsMemo, OptionsListUtils.getIOUConfirmationOptionsFromPayeePersonalDetail(payeePersonalDetailsMemo, String(myIOUAmount))];
+    }, [hasMultipleParticipants, selectedParticipantsMemo, iouAmount, iouCurrencyCode, payeePersonalDetailsMemo]);
+
+    useEffect(() => {
+        if (!isDistanceRequest) {
+            return;
+        }
+        const distanceMerchant = DistanceRequestUtils.getDistanceMerchant(
+            hasRoute,
+            distance,
+            mileageRate?.unit ?? CONST.CUSTOM_UNITS.DISTANCE_UNIT_MILES,
+            mileageRate?.rate ?? 0,
+            mileageRate?.currency ?? 'USD',
+            translate,
+            toLocaleDigit,
+        );
+        IOU.setMoneyRequestMerchant_temporaryForRefactor(transactionID, distanceMerchant);
+    }, [hasRoute, distance, mileageRate?.unit, mileageRate?.rate, mileageRate?.currency, translate, toLocaleDigit, isDistanceRequest, transactionID]);
+
+    const selectParticipant = useCallback(
+        (option: Participant) => {
+            // Return early if selected option is currently logged in user.
+            if (option.accountID === session?.accountID) {
+                return;
+            }
+            onSelectParticipant(option);
+        },
+        [session?.accountID, onSelectParticipant],
+    );
+
+    /**
+     * Navigate to report details or profile of selected user
+     */
+    const navigateToReportOrUserDetail = (option: Participant | OnyxTypes.Report) => {
+        if ('accountID' in option) {
```

Comment:
It looks like you changed the old logic
```suggestion
        if ('accountID' in option && option.accountID) {
```